javascript/webpack.config.devserver.js

```diff
@@ -47,7 +47,13 @@ module.exports = {
     })
   ],
   devServer: {
-    contentBase: path.join(__dirname, "dist"),
+    static: path.join(__dirname, "dist"),
```

Comment:
Did you intend to have this change in this PR?

Reply:
I had to fix this in order to get the JS visualizer to work (I guess Webpack silently changed in non-backward-compatible ways... >:( JS people...) and I was using the visualizer to debug. I guess if we were really being grown ups it'd be a separate PR, but that's a hassle so let's merge it in this one, if that's ok :)

Reply:
Totally fine with me, just checking!